packages/plugins/model-define/cycle-gan-model-define/src/index.ts

```diff
@@ -0,0 +1,145 @@
+import { ModelDefineType, UniModel, ModelDefineArgsType, ImageSample, ImageDataset } from '@pipcook/pipcook-core';
+import * as path from 'path';
+
+const boa = require('@pipcook/boa');
+const { tuple } = boa.builtins();
+const sys = boa.import('sys');
+sys.path.insert(0, path.join(__dirname, '..'));
```

Comment:
This is because boa doesn't support relative search? It's good to add a reasoned comment.